app/presenters/tree_node/storage.rb

```diff
@@ -1,4 +1,5 @@
 module TreeNode
   class Storage < Node
+    set_attribute(:text, &:name) # Even this is obvious, it's needed for additional changes in the C&U tree
```

Comment:
This is a change from `<stong>$name</strong> [$location]`, right?

Reply:
Yes and no, I need to set this so I can manipulate it in `override`. Everywhere else the Storage node requires just the `name`, which is the default behavior for any non-hash node. But to be able to `override` an attribute, it has to be defined here, even though it's default.